pkg/arvo/app/azimuth-tracker.hoon

```diff
@@ -40,13 +38,16 @@
 ::
 =>  |%
     ++  topics
-      =>  azimuth-events:azimuth
-      :_  ~
-      :~  broke-continuity
-          changed-keys
-          lost-sponsor
-          escape-accepted
-      ==
+      |=  ships=(set ship)
+      :-  =>  azimuth-events:azimuth
+          :~  broke-continuity
+              changed-keys
+              lost-sponsor
+              escape-accepted
+          ==
+      ?:  =(~ ships)
+        ~
+      [(turn ~(tap in ships) ,@) ~]
```

Comment:
The last few lines of this function look a bit funny.  A return type annotation would be helpful.

Reply:
Added return type and a couple comments